priam/src/main/java/com/netflix/priam/scheduler/SchedulerType.java

```diff
@@ -0,0 +1,31 @@
+package com.netflix.priam.scheduler;
+
+/**
+ * Created by aagrawal on 3/8/17.
+ */
+public enum SchedulerType {
+    TIME("TIME"),CRON("CRON");
```

Comment:
I think even flush implementation can be migrated to this format. `TIME` does not ring a bell that the values are expected in `HOUR` format. Even if we keep Flush implementation as is, these 2 formats are actually in sync - For e.g., Flush format  - `hour=0 or daily=10` which is in line with `SchedulerType=HOUR and backupHour =10.`